CHANGELOG.latest.md

```diff
@@ -1,7 +1,39 @@
-### Bugfixes
-* Fix entitlment info to string plan identifier (#906) via Josh Holtz (@joshdholtz)
-### Other Changes
-* Merge main -> bc5-support 2023-03-25 (#905) via Toni Rico (@tonidero)
-* Small improvements to v6-MIGRATION.md (#907) via Josh Holtz (@joshdholtz)
-* Make personalized price checkbox in PurchaseTester (#903) via swehner (@swehner)
-* PurchaseTester gets observable mode for google (#902) via Josh Holtz (@joshdholtz)
+**RevenueCat Android SDK v6** is here!! 😻 
+
+This latest release updates the SDK to use BillingClient 5. This version of BillingClient brings an entire new subscription model which has resulted large changes across the entire SDK.
+
+## Migration Guides
+- See the full list of API changes in [our v6 migration doc](https://github.com/RevenueCat/purchases-android/blob/v6/migrations/v6-MIGRATION.md)
+
+- See [Android Native - 5.x to 6.x Migration](https://www.revenuecat.com/docs/android-native-5x-to-6x-migration) for a 
+more thorough explanation of the new Google subscription model announced with BillingClient 5 and how to take advantage of them in V6.
+
+**Note:** This release is based off of `6.0.0-rc.1`. Developers migrating from that version shouldn't see any changes. 
+
+## API changes:
+There have been a lot of changes since v5! 
+
+Here are the highlights:
+
+### Purchasing API changes
+
+Purchases are now configured using `PurchaseParams.Builder()`. The builder is constructed with either a `Package`,
+`Product`, or `SubscriptionOption` and an `Activity`. The builder then has methods for setting the product change
+parameters (`oldProductId` and `googleProrationMode`).
+
+To initiate a purchase, simply pass the built `PurchaseParams` and your `PurchaseCallback` to the `purchase()` method.
+
+### New `SubscriptionOption` concept
+
+In V5, a purchase of a `Package` or `StoreProduct` represented a single purchaseable entity, and free trials or intro
+offers would automatically be applied if the user was eligible.
+
+Now, in v6, a `Package` or `StoreProduct` could contain multiple offers along within its base plan. 
```

Comment:
maybe we can call out here something like "a `Package` or `StoreProduct` represents a duration of a subscription, and could contain multiple offers along with its base plan`

the "within" should be "with" too